Scripts/generate_processor_docs.py

```diff
@@ -98,18 +99,22 @@ def indent_length(line_str):
 
 def main(_):
     """Do it all"""
-    usage = """%prog VERSION
+    usage = dedent("""%prog VERSION
 
-..where VERSION is the release version for which docs are being generated."""
+    ..where VERSION is the release version for which docs are being generated.""")
```

Comment:
Random comment: seems insane to have to import some random module to undo the consequences of slavishly adhering to a specific formatting style...